xmake/modules/core/tools/clang.lua

```diff
@@ -243,14 +243,18 @@ function nf_runtime(self, runtime, opt)
             maps["c++_shared"]    = "-stdlib=libc++"
             maps["stdc++_static"] = "-stdlib=libstdc++"
             maps["stdc++_shared"] = "-stdlib=libstdc++"
-            -- clang on windows fail to add libc++ includepath when using -stdlib=libc++ so we manually add it
-            -- @see https://github.com/llvm/llvm-project/issues/79647
-            if self:is_plat("windows") then
-                local llvm_rootdir = _get_llvm_rootdir(self)
-                if llvm_rootdir then
-                    maps["c++_static"] = table.join(maps["c++_static"], "-cxx-isystem" .. path.join(llvm_rootdir, "include", "c++", "v1"))
-                    maps["c++_shared"] = table.join(maps["c++_shared"], "-cxx-isystem" .. path.join(llvm_rootdir, "include", "c++", "v1"))
-                end
+            local llvm_rootdir
+            if get_config("sdk") then
```

Comment:
but it does not support `toolchain/set_sdkdir`

Reply:
i think it's okay now, can u confirm ? (i'm not sure how to get toolchain sdkdir)